src/pages/resources/recipes/network/index.md

```diff
@@ -0,0 +1,176 @@
+
+
+# Network I/O
+
+You may be looking for ways to make network calls for various reasons such as fetching an image, accessing a web API, browsing remote resources, synchronizing content, etc. UXP supports a range of network APIs that can fit your use case. 
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In plugins**, you should seek permission for `network` access in the manifest and specify the `domains` of the endpoints. <br></br> 
+IMPORTANT: Be sure you know about the [manifest permissions](../../../plugins/concepts/manifest/#permissionsdefinition) module before you proceed.
+
+**In scripts**, assume this value to be ''//TODO get value for scripts and ignore the manifest details below.
+
+
+// TODO Q: is localhost and http (macOS) allowed?
```

Comment:
@kareemmyk  Q: While configuring 'domains' in manifest.json, is localhost and http allowed in macOS?

Reply:
Shouldnt this be OS based rather than application based?